mlos_core/mlos_core/tests/spaces/adapters/llamatune_test.py

```diff
@@ -115,29 +122,30 @@ def test_special_parameter_values_validation() -> None:
     # Only UniformIntegerHyperparameters are currently supported
     with pytest.raises(NotImplementedError):
         special_param_values_dict_1 = {'str': 'choice_1'}
-        _ = LlamaTuneAdapter(
-            input_space,
+        LlamaTuneAdapter(
+            orig_parameter_space=input_space,
             num_low_dims=2,
             special_param_values=special_param_values_dict_1,
-            max_unique_values_per_param=None
+            max_unique_values_per_param=None,
         )
+
     with pytest.raises(NotImplementedError):
         special_param_values_dict_2 = {'cont': -1}
-        _ = LlamaTuneAdapter(
-            input_space,
+        LlamaTuneAdapter(
+            orig_parameter_space=input_space,
             num_low_dims=2,
             special_param_values=special_param_values_dict_2,
-            max_unique_values_per_param=None
+            max_unique_values_per_param=None,
```

Comment:
Would be cool if we could be more consistent about trailing commas or not and maybe even enforce it somehow.
I personally like to use them everywhere it's allowed since it makes `yy P cW` style vi edits easier and leaves git diffs only the line that got added, but I understand not all languages or sites support that.

Reply:
Yep, I agree and I try to put trailing commas everywhere -- I just missed that one!